go/kbfs/libkbfs/favorites.go

```diff
@@ -28,11 +28,11 @@ const (
 	favoritesCacheExpirationTime     = time.Hour * 24 * 7 // one week
 	kbfsFavoritesCacheSubfolder      = "kbfs_favorites"
 	favoritesDiskCacheFilename       = "kbfsFavorites.leveldb"
-	favoritesDiskCacheVersion        = 1
-	favoritesDiskCacheStorageVersion = 1
+	favoritesDiskCacheVersion        = 2
```

Comment:
I don't _think_ you need a new storage version for these changes.  It seems like you're only adding data/fields, and in that case msgpack does the right thing and will just initialize those fields as empty until you overwrite them.

Reply:
Do you mean to resolve this conversation without changing this back to 1?

Reply:
no, I thought I'd changed it back..

Reply:
Oh, this version needs to stay incremented, because the cache has changed from a `map[Favorite]bool` to a `map[Favorite]favoriteData`.